ci/github/build_driver_container

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+build_driver_container () {
+      cd ci/kmm-kmod && make KERNEL_SRC_DIR="/usr/src/linux-headers-$(uname -r)" && \
+      docker build -t kmm-kmod:local --build-arg=KERNEL_VERSION=$(uname -r) . && \
+      minikube image load kmm-kmod:local 
```

Comment:
Here is a good place to use `minikube image build` as we don't have any Makefile target for `kmm-kmod`.

Reply:
Could not find a way to make `minikube image build` work with arguments. That's why I splitted it. Any suggestion on how to make it work will be much appreciated.

Reply:
Can we use the default image name/tag as suggested [here](https://github.com/kubernetes-sigs/kernel-module-management/pull/159#discussion_r1029168571)? In that case you don't need any argument.